src/main/java/eu/darkbot/api/managers/WindowAPI.java

```diff
@@ -0,0 +1,96 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+import eu.darkbot.api.entities.utils.Area;
+import eu.darkbot.api.objects.Point;
+
+/**
+ * Utility to manage game window
+ */
+public interface WindowAPI extends API {
+
+    /**
+     * @return bounds of game screen
+     */
+    Area.Rectangle getViewBounds();
+
+    /**
+     * @return version of native api
+     */
+    int getVersion();
+
+    /**
+     * Triggers reload
+     */
+    void reload();
+
+    /**
+     * Sets size of game API window
+     */
+    void setSize(int width, int height);
+
+    /**
+     * Makes browser window visible.
+     */
+    void setVisible(boolean visible);
+
+    /**
+     * Hides window, reduces cpu usage.
+     * For some is bugged
+     */
+    void setMinimized(boolean visible);
+
+    /**
+     * Sends string to game window
+     *
+     * @param text to send
+     */
+    void sendText(String text);
+
+    /**
+     * Simulates key press and release
+     *
+     * @param keyCode to send
+     */
+    void keyClick(int keyCode);
+
+    default void keyClick(Character keyCode) {
+        if (keyCode != null) keyClick(keyCode.charValue());
+    }
+
+    /**
+     * Moves mouse to x & y coordinates of game window
+     */
+    void mouseMove(double x, double y);  //cast to int here or in implementation?
```

Comment:
probably the param here should be int, since i don't think you can move a mouse to a double coordinate on screen